news/2025/12/05/ModelingToolkitv11.md

```diff
@@ -0,0 +1,271 @@
+year and a half. It is generally rather buggy and leads to more difficult debugging. Thus for around the last
+year, for most issues we have suggested that users rewrite their code to not use the macro whenever sharing
+bug reports. We are finally making the decision to fully deprecate the `@mtkmodel` macro. The reasoning is
+simply that there are almost no maintainers, so while it does something interesting, it is not at a state
+which gives ModelingToolkit.jl a good look, and thus should not be what people see in the first tutorial.
+
+Because of this, all of the documentation will be updated to instead use the direct ModelingToolkit.jl form.
+We believe that this accomplishes a few goals. One it decreases the feature surface, making it easier to have a
+less buggy project and thus simply a better user experience. It also more clearly brings users into the
+"core" usage of ModelingToolkit as a compiler, where the direct interface has always been more flexible for
+many use cases like building libraries and things which are not related specifically to acausal modeling of
+DAEs (for example, `@mtkmodel` still does not even support SDEs). This lets the ModelingToolkit.jl team focus
+on what ModelingToolkit does best, namely, it's a symbolic representation for models and a compiler to fast
+and stable numerical simulations. 
+
+That does not mean that we do not like DSLs on ModelingToolkit, oh not at all! Instead this is making `@mtkmodel`
+no longer a privledged DSL of the project. For example, SymBoltz.jl and Catalyst.jl are two great DSLs built on
+ModelingToolkit.jl, just as separate packages. We plan to spawn out `@mtkmodel` to a separate DSL package of
+ModelingToolkit.jl in a similar fashion, though with the code base currently having no maintainers we will 
+deprecate the package unless new maintainers come to the revive the project. This is thus an open call for new
+folks to come in.
+
+We also must note that if someone really does need a fully developed DSL that is Modelica-like and compiles to
+ModelingToolkit, but is a more fully featured version (for example, has syntax highlighting, gives syntactic
+compiler errors, supports units, is well-maintained, etc.) then we suggest checking out 
+[Dyad](https://help.juliahub.com/dyad/dev/) which is free for non-commerical use and provides exactly this DSL.
+Our intention here is not to push out `@mtkmodel` to Dyad, but instead to note how the ModelingToolkit
+project has evolved, where the open source aspect has really evolved to thrive around the symbolic compiler but
+not all of the extra language features. As such, we want to make `@mtkmodel` like every other DSL as just a
+potential add-on package, and allow users to pick from the bevy of cool DSLs built on ModelingToolkit, keeping
+all of the difficulties of developing a language separate from the largely mathematical work of ModelingToolkit.
```

Comment:
Do we want to link https://discourse.julialang.org/t/using-mtk-when-i-import-modelingtoolkit/133681/13 somewhere here for additional context?